data/personas/healthcare_mentor_amiable.yaml

```
@@ -0,0 +1,102 @@
+
+vocabulary_level: professional
+response_speed: moderate
+humor_level: 0.4
+
+# シナリオ別の振る舞い設定例
+scenario_behaviors:
+  new_staff_guidance:
+    stress_level: low
+    cooperation: nurturing
+    communication: encouraging
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)